### PR TITLE
 [CHK-3092] feat: new postman test api collection

### PR DESCRIPTION
**Pull request by EmanueleBVtech**
#### List of Changes

- Added postman api test collections
- Added postman api env files

#### Motivation and Context
The goal is to have a set of automated tests ready for our new set od payment refund operations

#### How Has This Been Tested?
Manual Test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by infantesimone** [comment]
How can I run these api tests locally?

**Comment by EmanueleBVtech** [comment]
> **Check PR / Check Size (pull_request) ** Failing after 6s
hello,
in the root folder of the project you may find the api-tests folder, inside there are the env folder and the v1 folder.

to run the local test in postman, import the collection "helpdeskcommands.api.tests.local.json" from the v1 folder and then import the env file "helpdeskcommands_local.env.json" from env folder, then set this env as the active env in postman.

once the project is running, you can run the api and see the result

Note that at the current moment the API is still in very early development and the response is a static mock.

**Comment by EmanueleBVtech** [comment]
> How can I run these api tests locally?

hello,
in the root folder of the project you may find the api-tests folder, inside there are the env folder and the v1 folder.

to run the local test in postman, import the collection "helpdeskcommands.api.tests.local.json" from the v1 folder and then import the env file "helpdeskcommands_local.env.json" from env folder, then set this env as the active env in postman.

once the project is running, you can run the api and see the result

Note that at the current moment the API is still in very early development and the response is a static mock.

**Comment by infantesimone** [comment]
> > How can I run these api tests locally?
> 
> hello, in the root folder of the project you may find the api-tests folder, inside there are the env folder and the v1 folder.
> 
> to run the local test in postman, import the collection "helpdeskcommands.api.tests.local.json" from the v1 folder and then import the env file "helpdeskcommands_local.env.json" from env folder, then set this env as the active env in postman.
> 
> once the project is running, you can run the api and see the result
> 
> Note that at the current moment the API is still in very early development and the response is a static mock.

ok, so api test local can be useful in code review pipeline

File: api-tests/env/helpdeskcommands_uat_blue.env.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
```
Comment (infantesimone) [inline api-tests/env/helpdeskcommands_uat_blue.env.json:1]: delete this file, for this first beta release we are not introducing blue green deployment

Comment (EmanueleBVtech) [reply api-tests/env/helpdeskcommands_uat_blue.env.json]: done 👍

**Review by infantesimone — APPROVED** [review]

**Pull request merged**